torchci/rockset/metrics/__sql/master_commit_red_percentage.sql

```diff
@@ -0,0 +1,71 @@
+with any_red as (
```

Comment:
Looks like we need to download it again?

Reply:
fixed